email/parser.go

```diff
@@ -93,6 +104,23 @@ func parseMessageWithHeader(headers Header, bodyReader io.Reader) (*Message, err
 	}, nil
 }
 
+// removeDuplicate removes duplicate from bogus content type preventing parsing of email
+func removeDuplicate(s string) string {
+	m := make(map[string]bool)
+	parts := strings.Split(s, ";")
```

Comment:
That's a good point, I'm reading this in the RFC 2046 

> In general, the top-level media type is used to declare the general
   type of data, while the subtype specifies a specific format for that
   type of data.

Not sure if "In General" means "required" though? I can have the first attribute saved, and remove duplicate than returning first attr + the cleaned up ones?